build.gradle

```diff
@@ -17,72 +15,13 @@ java {
     sourceCompatibility = JavaVersion.VERSION_21
 }
 
-// store name of method parameter names in .class files (used to build
```

Comment:
dieses snippet wurde von @malt-r eingebaut, scheinbar wird es für antlr oder die dsl benötigt (kann nicht behauptet, dass ich den kommentar verstehe). bitte in das dungeon-gradle übernehmen